profiler/profile.go

```diff
@@ -44,48 +44,125 @@ const (
 	MetricsProfile
 )
 
-func (t ProfileType) String() string {
-	switch t {
-	case HeapProfile:
-		return "heap"
-	case CPUProfile:
-		return "cpu"
-	case MutexProfile:
-		return "mutex"
-	case BlockProfile:
-		return "block"
-	case GoroutineProfile:
-		return "goroutine"
-	case expGoroutineWaitProfile:
-		return "goroutinewait"
-	case MetricsProfile:
-		return "metrics"
-	default:
-		return "unknown"
+// collector holds the implementation details of a ProfileType, see collectors
+// map below.
+type collector struct {
+	// Name is used to lookup the profile for pprof.Lookup(name) and returned by
+	// ProfileType.String(). For profile types that don't use this approach, the
+	// name can be chosen arbitrarily.
+	Name string
+	// Filename is the filename used for uploading the profile to the datadog
+	// backend which is aware of them. Delta profiles are prefixed with "delta-"
+	// automatically.
+	Filename string
+	// Delta enables delta profiling for pprof profiles. This is useful for
+	// profiles that are represent samples collected over the lifetime of the
```

Comment:
```suggestion
	// profiles that represent samples collected over the lifetime of the
```
There is some typo here, but not sure if this is it 🙂

Reply:
💯 definitely a typo, and I think this fixed it. Thanks!